docs/advanced_guide/cloud/notebooks/text_generation/text_generation.ipynb

```diff
@@ -0,0 +1,469 @@
+      "outputs": [],
+      "source": [
+        "output_name = \"output_dir\"\n",
+        "download_path = \"output\"\n",
+        "\n",
+        "aml_helper.download_job_output(ml_client, job_name=s_job.name, output_name=output_name, download_path=download_path)\n",
+        "\n",
+        "downloaded_folder = Path(download_path) / \"named-outputs\" / output_name"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "#### Show Pareto Frontiers"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "param_vs_latency_img = Image(filename=downloaded_folder / \"pareto_non_embedding_params_vs_onnx_latency.png\")\n",
+        "display(param_vs_latency_img)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "param_vs_memory_img = Image(filename=downloaded_folder / \"pareto_non_embedding_params_vs_onnx_memory.png\")\n",
+        "display(param_vs_memory_img)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "latency_vs_memory_img = Image(filename=downloaded_folder / \"pareto_onnx_latency_vs_onnx_memory.png\")\n",
+        "display(latency_vs_memory_img)"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "#### Show search state of the last iteration"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "df = nb_helper.get_search_csv(downloaded_folder)\n",
+        "csv_as_html = nb_helper.get_csv_as_stylized_html(df)\n",
+        "display(HTML(csv_as_html))"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "source": [
+        "### Job 2: Train (Train a Pareto architecture from Transformer-Flex.)"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "#### Pick an architecture id (archid) from the CSV file to perform full training on"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "archid = \"<arch-id>\"\n",
+        "arch_path = nb_helper.get_arch_abs_path(archid=archid, downloaded_folder=downloaded_folder)"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "#### Load the training job from a YAML file and set the arch_path as its input"
```

Comment:
why does this need to be a heading?  I'd remove the '####'

Reply:
Done.